src/pbview.cpp

```diff
@@ -102,6 +102,11 @@ void PbView::run(bool auto_download, bool wrap_scroll)
 				i++;
 			}
 
+			if (i >= 1 && dllist_form.get("msg").length() == 0) {
```

Comment:
What's the reasoning behind `dllist_form.get("msg").length() == 0` here? I'm not sure it's necessary to preserve the previous status, but you probably included this for a reason?

Reply:
yeah, I was debating about the logic for that if statement myself for quite a while. The problem is if I omit 
`dllist_form.get("msg").length() == 0`,  it will never display the error messages that are set in some switch
case branches as they'll always be replaced with the status message stored in the download object.

I wasn't really sure how else to go about this problem.